src/PointwiseFunctions/Xcts/CenterOfMass.cpp

```diff
@@ -6,20 +6,42 @@
 namespace Xcts {
 
 void center_of_mass_surface_integrand(
-    gsl::not_null<tnsr::I<DataVector, 3>*> result,
+    gsl::not_null<tnsr::II<DataVector, 3>*> result,
```

Comment:
I suggest you don't do this change, since you'll just contract with the unit normal again for the surface integral.

Reply:
I was thinking that it would be nice to keep the surface integrands consistent between each other. That is, wouldn't it be confusing if some integrands need to be contracted before taking the surface integral ($P_{ADM}$ and $M_{ADM}$) and this one doesn't? Or maybe it doesn't matter much because these integrands are probably not going to be used by other parts of the code?

Reply:
Nah, rather don't do unnecessary computation. You can write in the docs how the integrand is supposed to be integrated.